lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup_facts.py

```diff
@@ -171,6 +197,25 @@ def gather_dvs_portgroup_facts(self):
                         vlan_override=dvs_pg.config.policy.vlanOverrideAllowed
                     )
 
+                if self.params['show_vlan_info']:
+                    vlanInfo = dvs_pg.config.defaultPortConfig.vlan
+                    if isinstance(vlanInfo, vim.dvs.VmwareDistributedVirtualSwitch.TrunkVlanSpec):
+                        vlan_id_list = []
+                        for vli in vlanInfo.vlanId:
+                            if vli.start == vli.end:
+                                vlan_id_list.append(str(vli.start))
+                            else:
+                                vlan_id_list.append(str(vli.start) + "-" + str(vli.end))
+                        vlan_info = dict(
+                            trunk=True,
+                            vlan_id=vlan_id_list
+                        )
+                    else:
+                        vlan_info = dict(
+                            trunk=False,
+                            vlan_id=vlanInfo.vlanId
```

Comment:
I get 

```
AttributeError: 'vim.dvs.VmwareDistributedVirtualSwitch.PvlanSpec' object has no attribute 'vlanId'
```

Reply:
hmm, how did you manage to get the error? I'm using version 6.5 and portgroups with vlan "none" give me vlan id 0.

Any suggestions, very much appreciated.

Reply:
I am using Private VLAN configuration while creating the dvpg. 

<img width="935" alt="Screenshot 2019-07-12 at 11 31 26 AM" src="https://user-images.githubusercontent.com/633765/61105801-a2832a00-a498-11e9-9029-d2a1169ee65e.png">


So you might want to add 
```
if isinstance(vlanInfo, vim.dvs.VmwareDistributedVirtualSwitch.PvlanSpec)
```

Reply:
done, thanks for the hint. 
Added a new bool in the return dict:  pvlan.